docs/src/concepts.md

```diff
@@ -0,0 +1,61 @@
+# Concepts
+
+## Software interoperability
+
+While software versioning helps researchers keep track of changes over time, it often
+makes sense to separate the interface from the pipeline itself. By collecting utilities
+here in Element Interface, maintainers can keep up to date with the latest developments
+across other packages, without causing issues in the respective Element.
+
+## Element Features
+
+### General utilities
+
+`utils.find_full_path` and `utils.find_root_directory` are used 
+across many Elements and Workflows to allow for the flexibility of providing 
+one or more root directories in the user's config, and extrapolating from a relative
+path at runtime.
+
+`utils.ingest_csv_to_table` is used across didactic workflows to ingest from example
```

Comment:
```suggestion
`utils.ingest_csv_to_table` is used across workflow examples to ingest from sample data from 
```

Reply:
Sorry, I thought this was one of my element PRs. Feel free to revert if needed

Reply:
No need, I'm on board with this change